lib/erb_lint/runner_config.rb

```diff
@@ -45,24 +45,29 @@ def merge!(other_config)
     end
 
     class << self
-      def default
+      def default(defaults_enabled: true)
         new(
           linters: {
-            AllowedScriptType: { enabled: true },
-            ClosingErbTagIndent: { enabled: true },
-            ExtraNewline: { enabled: true },
-            FinalNewline: { enabled: true },
-            NoJavascriptTagHelper: { enabled: true },
-            ParserErrors: { enabled: true },
-            RightTrim: { enabled: true },
-            SelfClosingTag: { enabled: true },
-            SpaceAroundErbTag: { enabled: true },
-            SpaceIndentation: { enabled: true },
-            SpaceInHtmlTag: { enabled: true },
-            TrailingWhitespace: { enabled: true },
+            AllowedScriptType: { enabled: defaults_enabled },
+            ClosingErbTagIndent: { enabled: defaults_enabled },
+            ExtraNewline: { enabled: defaults_enabled },
+            FinalNewline: { enabled: defaults_enabled },
+            NoJavascriptTagHelper: { enabled: defaults_enabled },
+            ParserErrors: { enabled: defaults_enabled },
+            RightTrim: { enabled: defaults_enabled },
+            SelfClosingTag: { enabled: defaults_enabled },
+            SpaceAroundErbTag: { enabled: defaults_enabled },
+            SpaceIndentation: { enabled: defaults_enabled },
+            SpaceInHtmlTag: { enabled: defaults_enabled },
+            TrailingWhitespace: { enabled: defaults_enabled },
           },
         )
       end
+
+      def apply_defaults(config)
```

Comment:
nit: Since this is similar to `default`, perhaps we should name it similar. How about something like `default_for(config)`?

Another option: why not have `cli.rb` do this:

```ruby
default_enabled = config["linters"]["DefaultLintersEnabled"]
RunnerConfig.default(default_enabled: default_enabled)
```

which would drop the need for another method.

Reply:
We moved that bit from `cli.rb` into `runner_config.rb` because we didn't want `cli.rb` to care about the implementation logic of the defaults. 

👍 for the name change